.gitignore

```diff
@@ -1,5 +1,6 @@
 database.db
 *pyc
+books/migrations/*
```

Comment:
No worries, just making sure it was included for the right reasons!

I'd actually love to discuss some modifications to the models (in order to make them all at once, or at least the ones we see more essential), as it makes much sense to get them sorted out as soon as possible in order not to need revising future features once again when that's done. I'll open an issue!